webflow-designer-extension-cli/public/styles.css

```
@@ -343,89 +441,222 @@ select {
+.topbar-avatar {
+  width: var(--size--avatar);
+  height: var(--size--avatar);
+  border-radius: var(--radius--round);
+  background: var(--background-colour--avatar);
+  flex-shrink: 0;
+}
+
+/* ─── Objects: Buttons ───────────────────────────────────────────────────── */
+
+.btn {
+  appearance: button;
```

Stylelint (17.4.0)

[error] Expected "button" to be "auto" (declaration-property-value-keyword-no-deprecated)

(declaration-property-value-keyword-no-deprecated)